src/app/index.ts

```diff
@@ -1,2 +1,2 @@
 export * from './material-docs-app';
-export * from './app-module';
+export * from './app.module';
```

Comment:
@jelbourn prefers the `app-module` format.

Reply:
Really :( I'll change ASAP.